### PR TITLE
update(HTML): web/html/element/input/password

### PR DESCRIPTION
**Pull request by undead404**
Оригінальний вміст: [&lt;input type="password"&gt;@MDN](https://developer.mozilla.org/en-us/docs/Web/HTML/Element/input/password), [сирці &lt;input type="password"&gt;@GitHub](https://github.com/mdn/content/blob/main/files/en-us/web/html/element/input/password/index.md)

Нові зміни:
- [chore(learn): Move MDN Curriculum into Learning Area (#36967)](https://github.com/mdn/content/commit/5b20f5f4265f988f80f513db0e4b35c7e0cd70dc)
- [Remove "simple" part 4: change to "basic" (#36763)](https://github.com/mdn/content/commit/f10015d1752d5668d8fe0de29f9d9807de475d58)
- [HTML autocorrect attribute (#35692)](https://github.com/mdn/content/commit/709d3a56661f895e5b0a67ff969e381d503ddd45)

### CODE REVIEW

File: files/uk/web/html/element/input/password/index.md [hunk 1]
```diff
@@ -33,7 +33,10 @@ browser-compat: html.elements.input.type_password
 
 ## Додаткові атрибути
 
-На додачу до атрибутів, що діють на всіх елементах {{HTMLElement("input")}}, незалежно від їх типу, поля паролів приймають наступні.
+На додачу до [глобальних атрибутів](/uk/docs/Web/HTML/Global_attributes) та атрибутів, що працюють на всіх елементах {{HTMLElement("input")}}, незалежно від їхніх типів, поля телефонних номерів підтримують наступні атрибути.
+
+> [!NOTE]
+> Глобальний атрибут [`autocorrect`](/uk/docs/Web/HTML/Global_attributes/autocorrect) можна додати до полів паролю, проте доступне в ньому значення завжди буде `off`.
```
[the remark this answers is not in the document]
Comment (undead404) [reply files/uk/web/html/element/input/password/index.md:39]:
```suggestion
> Глобальний атрибут [`autocorrect`](/uk/docs/Web/HTML/Global_attributes/autocorrect) можна додати до полів пароля, проте доступне в ньому значення завжди буде `off`.
```

File: files/uk/web/html/element/input/password/index.md [hunk 2]
```diff
@@ -33,7 +33,10 @@ browser-compat: html.elements.input.type_password
 
 ## Додаткові атрибути
 
-На додачу до атрибутів, що діють на всіх елементах {{HTMLElement("input")}}, незалежно від їх типу, поля паролів приймають наступні.
+На додачу до [глобальних атрибутів](/uk/docs/Web/HTML/Global_attributes) та атрибутів, що працюють на всіх елементах {{HTMLElement("input")}}, незалежно від їхніх типів, поля телефонних номерів підтримують наступні атрибути.
```
Comment (alinkedd) [inline files/uk/web/html/element/input/password/index.md:36]:
```suggestion
На додачу до [глобальних атрибутів](/uk/docs/Web/HTML/Global_attributes) та атрибутів, що працюють на всіх елементах {{HTMLElement("input")}}, незалежно від їхніх типів, поля пароля підтримують наступні атрибути.
```

**Review by alinkedd — CHANGES_REQUESTED** [review]

File: files/uk/web/html/element/input/password/index.md (hunk 2 above)
Comment (alinkedd) [reply files/uk/web/html/element/input/password/index.md:36]: це забув

**Review by alinkedd — CHANGES_REQUESTED** [review]
забув оновити 235 рядок
https://github.com/mdn/content/commit/f10015d1752d5668d8fe0de29f9d9807de475d58#diff-caf6d80f1109a1dcb61dfdc0b86b162a2f98cdbecd5154e34fb21cc0e04197d2L235

**Review by alinkedd — APPROVED** [review]

**Review by AdriandeCita — APPROVED** [review]

**Pull request merged**